src/pages/products/[...id].tsx

```diff
@@ -0,0 +1,117 @@
+import { useState, useEffect } from 'react';
+import ControlBar from '@/lib/components/Core/ControlBar';
+import Logo from '@/lib/components/Core/Logo';
+import { Product } from '@/lib/types/product';
+import { useRouter } from 'next/router';
+import Image from 'next/image';
+import { Button } from '@mantine/core';
+
+function Products() {
+  const [product, setProduct] = useState<Product>();
+  const [clickedReadMore, setClickedReadMore] = useState(false);
+  const [orderCount, setOrderCount] = useState(1);
+  const router = useRouter();
+  const { id } = router.query;
+  const MAX_CHARACTERS = 135;
+  let isMaxCharacters = false;
+  useEffect(() => {
+    console.log(id);
+    if (id) {
+      fetch(`https://fakestoreapi.com/products/${id}`)
+        .then((res) => res.json())
+        .then((data) => setProduct(data));
+    }
+  }, [id]);
+  function addOrderCount() {
+    setOrderCount((oldCount) => oldCount + 1);
+  }
+  function subractOrderCount() {
+    if (orderCount > 1) {
+      setOrderCount((oldCount) => oldCount - 1);
+    }
+  }
+  if (product?.description && product?.description.length > MAX_CHARACTERS) {
+    isMaxCharacters = true;
+  }
+  console.log(product);
+  return (
+    <>
+      <ControlBar />
```

Comment:
You can remove these and just use the `Layout.tsx`. I have tested the product page with `Layout.tsx` and product isn't centered like it should be with Layout set. It needs to be fixed before it can be merged.